### PR TITLE
Fix bn and timestamp query in state fns

### PR DESCRIPTION
**Pull request by breathx**
By request of @LouiseMedova

### CODE REVIEW

File: examples/binaries/new-meta/state-v3/build.rs [hunk 1]
```diff
@@ -0,0 +1,21 @@
+// This file is part of Gear.
+
+// Copyright (C) 2021-2022 Gear Technologies Inc.
```
Comment (gshep) [inline examples/binaries/new-meta/state-v3/build.rs:3]: 2023

Comment (gshep) [reply examples/binaries/new-meta/state-v3/build.rs:3]: and for others too

File: examples/binaries/new-meta/state-v3/src/wasm.rs [hunk 2]
```diff
@@ -0,0 +1,22 @@
+use demo_meta_io::Wallet;
```
Comment (gshep) [inline examples/binaries/new-meta/state-v3/src/wasm.rs:1]: do we need license there?

**Review by gshep — APPROVED** [review]

File: examples/binaries/new-meta/state-v3/build.rs (hunk 1 above)
Comment (breathx) [reply examples/binaries/new-meta/state-v3/build.rs:3]: Let's change licenses in separate PR all over the project

**Pull request merged**